packages/@ionic/cli/src/commands/deploy/capacitor.ts

```diff
@@ -0,0 +1,66 @@
+import { BaseConfig } from "@ionic/cli-framework";
+import lodash = require("lodash");
```

Comment:
is `loadash` already a dependency?

Reply:
Yes, that's what's being used in the Capacitor 3 PR.